### ISSUE TITLE
How to specify certain slides in a split 

### ISSUE THREAD

**Issue by Arwa-AlRubaian** [ncoudray/DeepPATH#128]
Hello, 
I would like to specify the exact cases to be used in each of the train, valid, and test split. Is there a way to do so ? 

Also can I export the cases used for train/valid/ test like the files you provided for the binary STK11 classification. 

Thanks !

**Comment by ncoudray** [comment]
Hi - Yes, at step 0.2a (0d_SortTiles.py).
You would need to run it 3 times:
- once with --PercentTest=0 --PercentValid=0 and using your desired training set an input
- once with --PercentTest=100 --PercentValid=0 and using your desired test set an input
- once with --PercentTest=0 --PercentValid=100 and using your desired validation set an input

The different set can be either split in different folders when you tile them, or you can just provide the list as a text file (first column the slide ID, second its label) via the "JsonFile" option

HTH,
Best

**Comment by Arwa-AlRubaian** [comment]
Thanks !
